packages/transaction-controller/CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.0.0]
+### Added
+- Add `submittedTime` to transaction meta ([#1645](https://github.com/MetaMask/core/pull/1645))
+- Add support for actionId ([#1582](https://github.com/MetaMask/core/pull/1582))
+- Add name controller ([#1647](https://github.com/MetaMask/core/pull/1647))
+
+### Changed
+- **BREAKING**: Rename rawTransaction to rawTx in the persisted transaction metadata ([#1624](https://github.com/MetaMask/core/pull/1624))
+- Support confirming of external transactions in `TransactionController`  ([#1625](https://github.com/MetaMask/core/pull/1625))
```

Comment:
This is probably best in the `Added` section with a line such as:

`Add `confirmExternalTransaction` method.